src/neuroglancer/util/http_request.ts

```diff
@@ -134,6 +134,11 @@ export function parseSpecialUrl(url: string): string {
     let path = match[3];
     if (path === undefined) path = '';
     return `https://s3.amazonaws.com/${bucket}${path}`;
+  }  else if (protocol === 'matrix') {
+    const bucket = match[2];
+    let path = match[3];
+    if (path === undefined) path = '';
```

Comment:
Formatting. Use 
```
if (condition) {
   <statement>
}
```
If match[3] can only be undefined||string, I recommend using const path = match[3] || '', or a tertiary statement if it can be something other than undefined.

Additionally, why is bucket not checked for undefined?